Assets/LeapMotionModules/GraphicRenderer/Testing/Scripts/GraphicRendererTestBase.cs

```diff
@@ -0,0 +1,154 @@
+/******************************************************************************
+ * Copyright (C) Leap Motion, Inc. 2011-2017.                                 *
+ * Leap Motion proprietary and  confidential.                                 *
+ *                                                                            *
+ * Use subject to the terms of the Leap Motion SDK Agreement available at     *
+ * https://developer.leapmotion.com/sdk_agreement, or another agreement       *
+ * between Leap Motion and you, your company or other organization.           *
+ ******************************************************************************/
+
+using System;
+using NUnit.Framework;
```

Comment:
The tests folder breaks Graphic Renderer example builds currently. I think we need to put the Testing scripts behind an Editor folder?